File: Modul 3/py/PRAK305-2310817110007-MuhammadDaffaMusyafa.py
hari = 0
jam = 0
menit = 0
detik = 0
# kenapa sama dengan 0 karena jika tidak sama dengan 0 maka angkanya akan 625654522
inputvar = int(input())

if inputvar >= 86400:
    hari = inputvar // 86400
    inputvar %= 86400

if inputvar >= 3600:
    jam = inputvar // 3600
    inputvar %= 3600

if inputvar >= 60:
    menit = inputvar // 60
    inputvar %= 60

detik = inputvar

if hari > 0:
    print(f"{hari} hari:{jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: Modul 3/c/PRAK305-2310817110007-MuhammadDaffaMusyafa.c
#include <stdio.h>

int main(){
    int b;
    int hari =0, jam = 0, menit = 0, detik = 0;
    scanf("%d", &b);
    hari = b / 86400;
    b %= 86400;
    jam = b / 3600;
    b %= 3600;
    menit = b / 60;
    b %= 60;
    detik = b / 1;
    b %= 1;
    if (hari > 0)
    {
       printf("%d hari %02d:%02d:%02d:",hari , jam , menit , detik); 
    }
    else if (jam > 0)
    {
       printf("%02d:%02d:%02d:", jam , menit , detik); 
    }
    else if (menit > 0)
    {
       printf("%02d:%02d:%02d:", jam , menit , detik); 
    }
    else if (detik > 0)
    {
       printf("%02d:%02d:%02d:", jam , menit , detik); 
    }
    return 0;
}